packages/mobile/src/analytics/constants.ts

```diff
@@ -88,8 +88,19 @@ export enum CustomEventNames {
   invite_friends_sms = 'invite_friends_sms', // to count the # of taps on “Invite with SMS" button on Invite_Friends_Review
   invite_friends_whatsapp = 'invite_friends_whatsapp', // to count the # of taps on “Invite with WhatsApp" button on Invite_Friends_Review
 
+  backup_start = 'backup_start', // ‘set up now’ button click
```

Comment:
Nit: May be beyond the scope of this PR but it would be nice if we had a consistent comment format throughout this file (for example, "(count ... " or " Count quiz backspace pressed" etc)